drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -287,7 +290,16 @@ void RasterizerCanvasGLES3::canvas_render_items(RID p_to_render_target, Item *p_
 			texture_storage->render_target_copy_to_back_buffer(p_to_render_target, back_buffer_rect, backbuffer_gen_mipmaps);
 
 			backbuffer_copy = false;
-			material_screen_texture_found = true; //after a backbuffer copy, screen texture makes no further copies
+			backbuffer_gen_mipmaps = false;
+			material_screen_texture_cached = true; // After a backbuffer copy, screen texture makes no further copies.
+			material_screen_texture_mipmaps_cached = backbuffer_gen_mipmaps;
```

Comment:
Should this line be placed here or should it be moved above the `backbuffer_gen_mipmaps = false;` line above? I get the impression that it should be moved upwards so that `material_screen_texture_mipmaps_cached` is set to true if the above `render_target_copy_to_back_buffer()` call generated the mipmaps directly.